r/aoa_script.R

```diff
@@ -296,13 +330,27 @@ if(parameters$use_pretrained_model == "false") { #train model ig no pretrained m
   print("--> predictors set")
   response <- response #set response value
   print("--> response set")
-  model <- train(training_data[,predictors], training_data$class, #train model
-                  method="rf", tuneGrid=data.frame("mtry"= length(predictors)/2), #with random forrest 
-                  importance=TRUE,
-                  ntree=parameters$procedure$random_forrest$n_tree, #max number of trees
-                  trControl=trainControl(method="cv", number=parameters$procedure$random_forrest$cross_validation_folds)) #perform cross validation to assess model
-  print("--> model trained")
-  model
+  if(parameters$procedure$selected == "rf") {
```

Comment:
Ist das hier nicht eher die Einbindung einer Auswahlmöglichkeit beim MLA? Also ein anderes Issue?

Reply:
Das ist die Implementierung in R. Nicht das Einbinden ins Frontend. Es ist also das richtige Issue.